### ISSUE TITLE
Demo files not working on master

### ISSUE THREAD

**Issue by anuvarsh** [dondi/GRNsight#166]
Neither of the demo files on the home page are loading. I assume this is because of the test-file name changes made for #156.

Everything in beta is working fine, so just the demo file names in the master branch need to be reverted back to the previous version. I'm not sure how to do that right now, so I am assigning this to @NAnguiano for now.

**Comment by dondi** [comment]
I was able to replicate this issue, and when I looked at the error console it appears that the client tried to load this URL: `http://localhost:3000/demo/unweighted`. `localhost:3000` is the problem here; that should be `grnsight.cs.lmu.edu` (as the beta correctly does). I suspect there is a configuration issue in the current process; @NAnguiano might have run the production server under the `dev` environment rather than `production`.
